descriptors/ModuleDescriptor-template.json

```diff
@@ -1179,6 +1179,27 @@
         }
       ]
     },
+    {
+      "id": "orders.send-claims",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/orders/claim",
+          "permissionsRequired": [
+            "orders.send-claims.collection.post"
+          ],
+          "modulePermissions": [
+            "configuration.entries.collection.get",
+            "orders-storage.pieces.collection.get",
+            "orders-storage.pieces.item.put",
+            "orders-storage.po-lines.item.get",
+            "orders-storage.po-lines.item.put",
+            "orders-storage.purchase-orders.item.get"
+          ]
+        }
+      ]
+    },
```

Comment:
required or optional interface for integration with mod-data-export-spring should be added.
Need to what to choose with thinking about applications.
If we do it as required we will be needed to add mod-data-export-spring in orders application.
Thinking optional is good here, because orders functionality will be functional even without send claim feature.
Code should be ready to handle missing rest endpoint in this case and show appropriate error message.